docs/network-flow-visibility.md

```diff
@@ -831,8 +885,9 @@ Currently we only support the visualization of NetworkPolicies with `Allow` acti
 
 If you would like to make any changes to any of the pre-built dashboards, or build
 a new dashboard, please follow this [doc](https://grafana.com/docs/grafana/latest/dashboards/export-import/)
-for dashboard export and import. To generate a deployment manifest with the changes,
-please follow the following steps:
+for dashboard JSON files export and manual import.
+
+To generate a deployment manifest with the changes, please follow the following steps:
 
 1. Clone the repository. Exported dashboard JSON files should be placed under `antrea/build/yamls/flow-visibility/base/provisioning/dashboards`.
 1. If a new dashboard is added, edit [kustomization.yml][flow_visibility_kustomization_yaml]
```

Comment:
Should there be two number '1' items in this list?

Reply:
I was following the markdown syntax of creating numbered list without writing actual number. 
https://www.markdownguide.org/basic-syntax/#ordered-lists